ui/app/product-milestones/close-result/pncProductMilestoneCloseResultPage.js

```diff
@@ -43,12 +43,23 @@
 
       $ctrl.prefixFilters = 'loggerName.keyword:org.jboss.pnc.causeway|org.jboss.pnc._userlog_,level.keyword:INFO|ERROR|WARN';
       $ctrl.matchFilters = `mdc.processContext.keyword:${$ctrl.data.id}`;
+
+      $scope.$on('MILESTONE_PUSH_STATUS_CHANGE', (event, milestonePushResult) => {
+        $scope.$applyAsync(() => load(milestonePushResult));
+      });
     };
 
     function showTable() {
         return $ctrl.data.buildPushResults && $ctrl.data.buildPushResults.length;
     }
 
+    function load(milestonePushResult) {
+      if (milestonePushResult) {
+        $ctrl.data = milestonePushResult;
+        $ctrl.prefixFilters = 'loggerName.keyword:org.jboss.pnc.causeway|org.jboss.pnc._userlog_,level.keyword:INFO|ERROR|WARN';
```

Comment:
Is this changing? It could be omitted if not

Reply:
yes, you're right, thanks!